aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie60/ProgramSeed.cs

```diff
@@ -0,0 +1,104 @@
+#define DEFAULT // SQL server is default, SQL_Lite is other
+#if DEFAULT
+#region snippet_all
+using Microsoft.EntityFrameworkCore;
+using RazorPagesMovie.Data;
+using RazorPagesMovie.Models;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorPages();
+
+builder.Services.AddDbContext<RazorPagesMovieContext>(options => 
+       options.UseSqlServer(builder.Configuration.GetConnectionString("RazorPagesMovieContext")));
+
+var app = builder.Build();
+
+using (var scope = app.Services.CreateScope())
+{
+    var services = scope.ServiceProvider;
+
+    try
+    {
+        SeedData.Initialize(services);
+    }
+    catch (Exception ex)
+    {
+        var logger = services.GetRequiredService<ILogger<Movie>>();
+        logger.LogError(ex, "An error occurred seeding the DB.");
+    }
+}
```

Comment:
@halter73 is this the right place to call `SeedData.Initialize` ?

Reply:
Looks like the place to me, but would it be better just to let the program crash at startup if the seeding fails? You can remove the call to UseDeveloperExceptionPage() here as well.

Reply:
> Looks like the place to me, but would it be better just to let the program crash at startup if the seeding fails? 
Yes, I'll do that

> You can remove the call to UseDeveloperExceptionPage() here as well.

I can't for this tutorial because the code needs to match what the templates generate. I did open https://github.com/dotnet/Scaffolding/issues/1629 asking that ` UseDeveloperExceptionPage()` be removed from the templates.
Muchos Gracious again :)